.babelrc

```diff
@@ -1,10 +1,19 @@
 {
-  "presets": [
+  "presets": ["@babel/preset-env"],
+  "plugins": [
     [
-      "@babel/preset-env",
+      "module-resolver",
       {
-        "targets": {
-          "node": "current"
+        "root": ["./src"],
+        "alias": {
+          "@": "./src",
+          "components": "./src/components",
+          "middlewares": "./src/middlewares",
+          "services": "./src/services",
+          "models": "./src/models",
+          "utils": "./src/utils",
+          "validators": "./src/validators",
+          "constants": "./src/constants"
```

Comment:
Let's prefix it with `@`, e.g: `@middlewares`

Reply:
okay let's do this way